piety-promoted-v3/modernized/01-chapter-01.adoc

```diff
@@ -140,7 +140,7 @@ O the elders! the elders! they should dig for the arising
 of the well of life as with the staves in their hands.
 Spring up, O well, and I will sing unto you!`' At another time,
 in a manner similar to this, he uttered these words,
-Those that are faithful to the end shall receive a crown, a crown that fades not away;
+"`Those that are faithful to the end shall receive a crown, a crown that fades not away;
```

Comment:
I think to match the ending quotation, this should be a single-quote, not a double.

Reply:
Sounds good, that was a typo. :)